pkg/controller/add_node.go

```diff
@@ -0,0 +1,10 @@
+package controller
```

Comment:
Did you add this manually or run some SDK command?

Reply:
I used the sdk command to add a controller

Reply:
Please add the SDK command to your commit message.